Dockerfile.enhanced

```diff
@@ -4,19 +4,19 @@ WORKDIR /home/docsworker-xlarge
 COPY  config config/
 COPY package*.json ./
 COPY tsconfig*.json ./
-RUN npm install
+RUN npm ci --legacy-peer-deps
```

Comment:
minor. this is only required for snooty and should be mitigated by @rayangler 's [PR for npmrc ](https://github.com/mongodb/snooty/pull/842)